### PR TITLE
Add top nav

### PR DESCRIPTION
**Pull request by rifont**
**What?**
* Add Top NavBar through custom JS

**Why?**
* We want to support links to other websites within the Novu ecosystem

**Screenshots**

_Light_
image

_Dark_
image

### CODE REVIEW

File: mint.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "$schema": "https://mintlify.com/schema.json",
   "name": "Novu",
   "logo": {
-    "dark": "/logo/logo-dark-bg.svg",
-    "light": "/logo/logo-light-bg.svg"
+    "dark": "/logo/novu-logo-placeholder.svg",
+    "light": "/logo/novu-logo-placeholder.svg"
   },
-  "openapi": "https://api.novu.co/api-json",
+  "openapi": "https://api.novu.co/api-json?utm_campaign=docs-openapi",
```
Comment (rifont) [inline mint.json:8]: This one needs to be machine readable for the Mintlify crawler, I'll revert and update with the non-deprecated endpoint (https://api.novu.co/openapi.json).

Comment (rifont) [inline mint.json:8]:
```suggestion
  "openapi": "https://api.novu.co/openapi.json",
```

File: mint.json [hunk 2]
```diff
@@ -636,7 +636,7 @@
     },
     {
       "group": "Getting Started",
-      "pages": ["echo/introduction", "echo/quickstart", "echo/features/studio"],
+      "pages": ["echo/introduction", "echo/quickstart", "echo/concepts/studio"],
```
Comment (rifont) [inline mint.json:639]: This one was broken.

**Comment by justnems** [comment]
Looks great. Let's let her rip, and see what happens!

**Review by justnems — APPROVED** [review]
Looks good. I changed one link. Approved, and let's see what happens when it's released!

**Pull request merged**